comp/rdnsquerier/impl-none/none.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+// Package rdnsquerierimplnone provides the noop rdnsquerier component
+package rdnsquerierimplnone
+
+import (
+	rdnsquerier "github.com/DataDog/datadog-agent/comp/rdnsquerier/def"
+)
+
+// Provides defines the output of the rdnsquerier component
+type Provides struct {
```

Comment:
Should this extend `fx.Out`?

Reply:
I don't think that is needed anymore, it's not mentioned in https://datadoghq.dev/datadog-agent/components/creating-components/.